### ISSUE TITLE
[Feature Request] Last seen complete [PR Welcome]

### ISSUE THREAD

**Issue by hvalev** [ManiMatter/decluttarr#211]
Hi, great tool! Thank you for making it.

I'd like to propose utilizing the last seen complete attribute to remove ghost torrents. It could work by checking whether there have been X number of days until it has been seen as complete and if that duration has been exceeded the torrent is removed.

**Comment by ManiMatter** [comment]
Good idea. Im limited on time, would you like to contribute with a PR?

**Comment by ManiMatter** [comment]
closing as not planned. happy to review a pr if you want to code this.
